static/window-template.tmpl

```diff
@@ -0,0 +1,333 @@
+<!doctype html>
+<html lang="en-us">
+    <head>
+        <title>CPython</title>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <!-- <base href="{{cookiecutter.cdn}}" target="_blank"> -->
+        <!-- <link rel="icon" type="image/png" href="favicon.png" sizes="16x16"> -->
+
+        <link rel="stylesheet" href="{{cookiecutter.cdn}}/styles.css"/>
+
+        <link rel="stylesheet" href="{{cookiecutter.cdn}}/xtermjsixel/xterm.css" />
+        <link rel="stylesheet" href="{{cookiecutter.cdn}}/xtermjsixel/style.css"/>
+        <style>
+            .hidden {

```

Comment:
i'd prefer using the [html](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) tag hidden it's simpler and shorter ( that skips the  class= everywhere )

Reply:
oo i didn't know bout tht, ill edit tht too